### ISSUE TITLE
Training for HDR

### ISSUE THREAD

**Issue by IanMaquignaz** [Hans1984/GlowGAN#5]
Hi, 

I'm trying to train your model against our HDR dataset for a baseline comparison, but I am unsure of whether I have configured everything correctly. 

Our Dataloader:
* Images: RGB images up to [3,512,512] resolution; tonemapped to Gamma 2.2 with range [0,1]
* Labels: None, therefore all integer values of 0
* Augmentation: Our dataloader handles dataset-specific augmentation.

Training steps: 
Is it just this?
```bash
 python train.py --outdir=<OUT_DIR> --cfg=stylegan3-t -- 
  data=<DATA_DIR> --gpus=4 --batch=32 --mirror=1 --snap 20 --batch-gpu 8 --kimg 10000 --syn_layers 7
```

You have notes for super-resolution but its unclear how it should be applied. Would I be correct that the initial training resolution should be 256x256, then supper-resolution training with the images at 512x512 resolution?
```bash
python train.py --outdir=<OUT_UPSCALE_DIR> --cfg=stylegan3-t --data=<DATA_HIGH_RES_DIR> \
  --gpus=4 --batch=32 --mirror=1 --snap 20 --batch-gpu 8 --kimg 10000 --syn_layers 7 \
  --superres --up_factor 2 --head_layers 7 \
  --path_stem <PRETRAINED_MODEL_DIR>
```

That being said... our dataset is a calibrated collection of HDR images and therefore has no variations in exposure. We could 'augment' the dataset by adding exposure variation. 

Thank you in advance
